tests/unit/linalg/test_matrix_dense.cpp

```diff
@@ -15,6 +15,25 @@
 
 using namespace mfem;
 
+TEST_CASE("DenseMatrix init-list construction", "[DenseMatrix]")
+{
+   double ContigData[6] = {6.0, 5.0,
+                           4.0, 3.0,
+                           2.0, 1.0
+                          };
```

Comment:
```suggestion
   double ContigData[6] = {6.0, 5.0, 4.0, 3.0, 2.0, 1.0};
```
Currently it almost looks like the transpose (since column major). Might be less confusing if it's just like this